cluster-endpoints/src/grpc/gprc_accounts_streaming.rs

```diff
@@ -0,0 +1,215 @@
+                                    }
+                                }
+                                AccountFilterType::Memcmp(memcmp) => {
+                                    SubscribeRequestFilterAccountsFilter {
+                                        filter: Some(Filter::Memcmp(
+                                            SubscribeRequestFilterAccountsFilterMemcmp {
+                                                offset: memcmp.offset,
+                                                data: Some(match &memcmp.data {
+                                                    MemcmpFilterData::Bytes(bytes) => {
+                                                        Data::Bytes(bytes.clone())
+                                                    }
+                                                    MemcmpFilterData::Base58(data) => {
+                                                        Data::Base58(data.clone())
+                                                    }
+                                                    MemcmpFilterData::Base64(data) => {
+                                                        Data::Base64(data.clone())
+                                                    }
+                                                }),
+                                            },
+                                        )),
+                                    }
+                                }
+                            })
+                            .collect_vec()
+                    } else {
+                        vec![]
+                    };
+                    subscribe_accounts.insert(
+                        format!("accounts_{}", program_id),
+                        SubscribeRequestFilterAccounts {
+                            account: vec![],
+                            owner: vec![program_id.clone()],
+                            filters,
+                        },
+                    );
+                }
+            }
+
+            let mut stream = client
+                .subscribe_once(
+                    Default::default(),
+                    subscribe_accounts,
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
+                    Default::default(),
+                    Some(commitment),
+                    Default::default(),
+                    None,
+                )
+                .await?;
+
+            while let Some(message) = stream.next().await {
+                let Ok(message) = message else {
+                    continue;
+                };
+
+                let Some(update) = message.update_oneof else {
+                    continue;
+                };
+
+                match update {
+                    UpdateOneof::Account(account) => {
+                        if let Some(account_data) = account.account {
+                            let account_pk_bytes: [u8; 32] = account_data
```

Comment:
consider different abbreviation for pubkey in `account_pk`; maybe `account_key` or `account_pubkey`